src/App.js

```diff
@@ -1,11 +1,51 @@
-import React from 'react';
+import React, {useEffect, useState} from "react";
+import axios from "axios";
 import './App.css';
+import './components/PokemonCard';
+import PokemonCard from "./components/PokemonCard";
 
 function App() {
+    const [endpoint, setEndpoint] = useState(['https://pokeapi.co/api/v2/pokemon'])
+    const [pokemonData, setpokemonData] = useState([]);
+
+    useEffect(() => {
+        async function fetchPokemonData()
+        {
+            try
+            {
+                const pokemonData = await axios.get(`${endpoint}`);
```

Comment:
Hier kun je het iets versimpelen, omdat je de string niet verder uitbreidt dan `endpoint` zelf
```suggestion
                const pokemonData = await axios.get(endpoint);
```